requirements_test.txt

```diff
@@ -3,8 +3,8 @@ docopt
 psutil
 requests
 ansible
-ansible-core==2.16; python_version >= "3.11"
-ansible-core==2.15; python_version < "3.11"
+ansible-core>=2.16,<2.17; python_version >= "3.11"
+ansible-core>=2.15,<2.16; python_version < "3.11"
```

Comment:
I think that you can simplify them using `~=`, e.g.:
```
ansible-core~=2.16.0; python_version >= "3.11"
```
Reference: https://packaging.python.org/en/latest/specifications/version-specifiers/#version-specifiers